cmd/minikube/cmd/start.go

```diff
@@ -127,28 +128,34 @@ func runStart(cmd *cobra.Command, args []string) {
 		NetworkPlugin:     viper.GetString(networkPlugin),
 		ExtraOptions:      extraOptions,
 	}
+
+	fmt.Println("SSH-ing files into VM...")
```

Comment:
I think we should leave this out.  I think we want to de-emphasize the fact its running in a VM.  Maybe "Updating cluster..."?